spec/dor/permissable_spec.rb

```diff
@@ -0,0 +1,307 @@
+require 'spec_helper'
+
+describe Dor::Permissable do
```

Comment:
All new specs that can read as requirements from
https://consul.stanford.edu/display/chimera/Repository+Roles+and+Permissions